doc/src/manual/strings.md

```diff
@@ -228,6 +228,22 @@ julia> str[6:6]
 The former is a single character value of type `Char`, while the latter is a string value that
 happens to contain only a single character. In Julia these are very different things.
 
+It is possible to create a view into a string using the type `SubString`, for example:
+
+```jldoctest
+julia> str = "long string"
+"long string"
+
+julia> substr = SubString(str, 1, 4)
+"long"
+
+julia> typeof(substr)
+SubString{String}
+```
+
+`SubString` works like [`getindex`](@ref), but it does not make a copy of the parent string. Several
```

Comment:
Hmm, one problem is that `getindex` isn't presented for strings. Maybe just say that indexing in the examples presented above make a copy, but that one can also use `SubString`. Also, it would make more sense IMHO to say this before the examples: else, it's not clear what a "view" is nor why one would want to create a view.

Reply:
This is what I also thought after committing - I hope the current wording is clear. Sorry for the confusion.